src/diy/imu-comparison.md

```diff
@@ -46,6 +47,24 @@ Score: <i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"
 
 > Please note, if you are looking for BNO085s SlimeVR may have [spares for sale that can be found here](https://shop.slimevr.dev/products/slimevr-imu-module-bno085).
 
+---
+## LSM6DSV
+The LSM6DSV is a new and <b>experimental</b> IMU for DIY SlimeVR.
+Offers very good reset times even in bad magnetic enviroments and for fast movements, dancing. Low availability because the chip is new. 
```

Comment:
```suggestion
Can offer very good reset times even in bad magnetic enviroments and for fast movements, dancing. Low availability because the chip is new. 
```